presc/copies/sampling.py

```diff
@@ -296,6 +409,133 @@ def spherical_balancer_sampling(
     return df_generated
 
 
+def categorical_sampling(nsamples=500, random_state=None, feature_parameters=None):
```

Comment:
(here and below) if `feature_parameters` is actually required, it would be better to put it first in the param list with no default value, to make that clear.

Reply:
I changed this but then I realised that this also affects the `ClassifierCopy` class, which has a `**k_sampling_parameters` that includes the `feature_parameters` keyword argument, among others, and I am unsure how to proceed if this particular argument is then positional.

Should I add another argument (e.g. `*sampling_parameters`) to allow for the positional `feature_parameters` argument? Where exactly should I place this?

Maybe if this complicates things too much I could simply leave `feature_parameters` as a keyword argument, but add a note/error that the keyword argument is required?  :thinking:

Reply:
@dzeber, I tried to change the `feature_parameters` argument in all the sampling functions and leaving the `ClassifierCopy` class unchanged, as you suggested, but I get an error in the two tests involving the class that says "got multiple values for argument 'feature_parameters'" so there is something wrong.

Reply:
Looks like you still need to update where `mixed_data_sampling` is getting called.

Reply:
I've now finally understood and managed to fix what was wrong with the `mixed_data_sampling` function call within the `ClassifierCopy` class.  :)

As far as I can see, the new code is passing the tests of the ML Classifier Copying package. I think the CI fails for some other reason.